src/libs/actions/Card.ts

```diff
@@ -487,6 +489,44 @@ function updateExpensifyCardLimitType(workspaceAccountID: number, cardID: number
     API.write(WRITE_COMMANDS.UPDATE_EXPENSIFY_CARD_LIMIT_TYPE, parameters, {optimisticData, successData, failureData});
 }
 
+function deactivateCard(workspaceAccountID: number, cardID: number, card?: Card) {
+    const authToken = NetworkStore.getAuthToken();
+
+    if (!authToken) {
+        return;
+    }
```

Comment:
I think we can remove these, I am not sure sure why we started to add it, this should not happen
```suggestion
```

Reply:
@mountiny I think it's to narrow down the type of `authToken`, cause the type of `authToken` also includes undefined

Reply:
Urgh thats annoying